pandas/core/generic.py

```diff
@@ -3136,6 +3136,22 @@ def to_latex(
         if multirow is None:
             multirow = config.get_option("display.latex.multirow")
 
+        if is_list_like(formatters) and not isinstance(formatters, dict):
+            formatter_elems_type = all(
+                isinstance(elem, str) or callable(elem) for elem in formatters
+            )
+            if formatter_elems_type:
+                formatters = [
+                    (lambda style: lambda x: "{0:{1}}".format(x, style))(style)
```

Comment:
I would suggest not to use two folded lambda functions. This is not easily readable. Can it be replaced with an f-string?

Reply:
Initially they were simpler lambda functions to take advantage of the existing callable implementation, but there was a python memory specific error where it created the first anonymous function and replicated it multiple times. 

I believe this is using fstrings? At least for the substitution.  But yeah I agree it's a little hard to read.